library/oneview_id_pools_ipv4_range.py

```diff
@@ -71,10 +71,10 @@
     type: dict
 '''
```

Comment:
update copyright years for all the files. 
Change python, ansible and hpeOneView version to 3.4.2, 2.9, 6.0.0

Reply:
do you want to add these versions as minimum system requirements?
currently we have 
requirements:
    - "python >= 2.7.9"
    - "hpeOneView >= 4.0.0"